File: backend/src/main/scala/kpn/api/common/subset/SubsetOrphanRoutesPage.scala
package kpn.api.common.subset

import kpn.api.common.OrphanRouteInfo
import kpn.api.common.TimeInfo

case class SubsetOrphanRoutesPage(
  timeInfo: TimeInfo,
  subsetInfo: SubsetInfo,
  routes: Seq[OrphanRouteInfo]
)

File: backend/api/src/main/java/kpn/api/common/subset/SubsetOrphanRoutesPage.java
package kpn.api.common.subset;

import kpn.api.common.OrphanRouteInfo;
import kpn.api.common.TimeInfo;
import kpn.api.common.subset.SubsetInfo;

import com.google.common.collect.ImmutableList;

public record SubsetOrphanRoutesPage(
  TimeInfo timeInfo,
  SubsetInfo subsetInfo,
  ImmutableList<OrphanRouteInfo> routes
) {}
